wiki/Rules/el.md

```diff
@@ -0,0 +1,79 @@
+
+Off-Topic Κανόνες
+-----------------
+
+1.  Στο Off-Topic είναι εκεί που "πεθαίνουν" τα "κακά" threads. Τίποτα αξιοσημείωτο δεν βρίσκεται σε αυτό το subforum. Αν το θέμα σας έχει κάποια αξία, τότε δεν ανήκει στο off-topic. Αν το θέμα σας μεταφέρθηκε στο off-topic από κάποιο άλλο μέρος, σημαίνει πως έχει κριθεί ανάξιο σοβαρής συζήτησης.
+2.  Η υπερβολική χαζομάρα απαγορεύεται σε μεγάλο βαθμό. Εδώ εφαρμόζεται ο κανόνας \#3 της κοινότητας. Είναι εντάξει να αστειεύεσται από 'δώ και από 'κεί - αλλά όχι να γράφετε συνέχεια ανοησίες.
+3.  Τα threads γενεθλίων επιτρέπονται ΜΟΝΟ στο Off-topic, και πουθενά αλλού.
+
+In-game Chat Κανόνες
+--------------------
+
+1.  Απαγορεύεται το spam.
+2.  Σε περίπτωση που βρεθείτε να προσπαθείτε να αποφύγετε silences, θα οδηγηθείτε άμεσα σε περιορισμό, και θα βρεθείτε.
+    1.  Το να μιλάτε για silences (δικό σας προηγούμενο ή κάποιου άλλου) σε δημόσιες συζητήσεις απαγορεύεται και τιμωρείται με silence.
+3.  Δημόσια συζήτηση για NSFW περιεχόμενο ή οτιδήποτε σεξουαλικό δεν επιτρέπεται σύμφωνα με το κανόνα #5 της κοινότητας. Αυτός ο κανόνας δεν ισχύει για τα προσωπικά μηνύματα.
+4.  Όλοι οι σύνδεσμοι (links) με εξωτερικό περιεχόμενο (εικόνες, threads, βίντεο κ.λπ.) δεν πρέπει να περιέχουν φωτοευαίσθητο περιεχόμενο (γρήγορο φλας κ.λπ.).
+5.  Η πλαστοπροσωπία απαγορεύεται ρητά, ειδικά του προσωπικού (Staff).
+6.  Μη διαφημίζετε multiplayer δωμάτια πέρα από το #lobby.
+7.  Απαγορεύονται ρητά οι συζητήσεις περί cheating, hacking, ή οποιαδήποτε άλλη μορφή κακομεταχείρισης του client.
+    1.  Απαγορεύεται επίσης η αναφορά και η συζήτηση απαγορευμένων (banned) παικτών.
+8.  Σε κανάλια συγκεκριμένων γλωσσών, μιλήστε στην κυρίαρχη γλώσσα εάν συμμετέχετε σε μία συζήτηση. Όταν σε ένα κανάλι δεν αναφέρεται μια συγκεκριμένη γλώσσα, τότε είναι πάντα τα Αγγλικά.
+    1.  Οι Συντονιστές (moderators) μπορούν να μιλήσουν Αγγλικά σε κανάλια συγκεκριμένων γλωσσών, σε περίπτωση που κριθεί απαραίτητο, για να χειριστούν μια κατάσταση κατάλληλα.
+    2.  Σε όλα τα άλλα κανάλια (#osu, #english, #modhelp, #modreqs, #videogames, #ctb, #taiko, #osumania), απαγορεύεται να μιλάτε οποιαδήποτε άλλη γλώσσα εκτός από τα Αγγλικά.
+    3.  Τα κανάλια #lobby και #help είναι πολύγλωσσα, επομένως δεν υπάρχουν περιορισμοί στην γλώσσα.
+
+Client Κανόνες
+--------------
+
+1.  Η χειραγώγηση ή αλλοίωση του client, όπως το score hacking, macroing, που επηρεάζουν το χρονοδιάγραμμα στο οποίο δραστηριοποιείται το client, auto-aim και οτιδήποτε αφαιρεί το ανθρώπινο στοιχείο από το παιχνίδι θεωρείται cheating. Αν πρέπει να ρωτήσετε αν κάτι θεωρείται cheating, και χρησιμοποιήτε ένα εξωτερικό πρόγραμμα για να επιτευχθεί αυτό το αποτέλεσμα, πολύ πιθανόν να είναι.
+2.  Η αλλαγή στοιχείων του παιχνιδιού μέσω οποιασδήποτε μεθόδου ΕΚΤΟΣ από τα beatmap skins/client skins θεωρείται ως πιθανή εξαπάτηση και θα αντιμετωπίζεται αυστηρά.
```

Comment:
I prefer not to translate them. We use them in the English terminology amongst the Greek community. Also, the translations sound silly.